src/ws/session-utils.h

```diff
@@ -62,10 +62,6 @@ void build_string (char **buf, size_t *size, const char *str, size_t len);
 void authorize_logger (const char *data);
 void save_environment (void);
 void pass_to_child (int signo);
-#ifndef HAVE_FDWALK
-int fdwalk (int (*cb)(void *data, int fd), void *data);
-#endif
-int closefd (void *data, int fd);
```

Comment:
IMO we should keep the closing of all fds > 3. This is a security boundary, so let's not take chances from leaking fds to kerberos or other PAM services.

Reply:
Brought this back.